src/dotnet-serve/CommandLineOptions.cs

```diff
@@ -119,6 +119,9 @@ public virtual bool UseTls
         [Option("-z|--gzip", Description = "Enable gzip compression")]
         public bool UseGzip { get; }
 
+        [Option("-b|--brotli", Description = "Enable brotli compression")]
```

Comment:
```suggestion
#if !NETCOREAPP2_1
        [Option("-b|--brotli", Description = "Enable brotli compression")]
```

I think it would be better to hide this option altogether in netcoreapp2.1.